manifests/firewall.pp

```diff
@@ -10,11 +10,11 @@
 ) {
   if $manage_firewall {
     firewall { "100 allow port ${varnish_listen_port} to varnish":
-      chain  => 'INPUT',
-      proto  => 'tcp',
-      state  => ['NEW'],
-      dport  => $varnish_listen_port,
-      action => 'accept',
+      chain => 'INPUT',
+      proto => 'tcp',
+      state => ['NEW'],
+      dport => $varnish_listen_port,
+      jump  => 'accept',
```

Comment:
please do this in it's own PR. In addition we need to update the metadata.json to only allow the latest version where action got replaced with jump.

Reply:
ok, will do.